test/consumer.test.ts

```diff
@@ -0,0 +1,247 @@
+import { config, expect } from 'chai';
+import { SecurityEngine, DiagnosticsPipeline, codeActionsMap } from '../src/consumers';
+
+describe('Response consumer test', () => {
+    
+    it('Consume response for free-users', async () => {
+        let DiagnosticsEngines = [SecurityEngine];
+        const dependency = {
+            "name": { 
+                "value" : "abc",
+                "position": { 
+                    "line": 20,
+                    "column": 6
+                }
+            },
+            "version": {
+                "value": "1.2.3",
+                "position": {
+                    "line": 20,
+                    "column": 17
+                }
+            }
+        }
```

Comment:
This is repeated in all tests, can you move it to setup() and clear it off in teardown? or even global variable is fine